frontend/src/assets/css/variables.scss

```diff
@@ -11,5 +15,15 @@
   --violet-100: #e5e1ff;
   --pink-200: #e1a7d5;
   --red-300: #d51a52;
+  --white-alpha-20: #ffffff33;
+  --white-alpha-30: #ffffff4d;
   --white-alpha-80: #ffffff80;
+  --grey-alpha-25: #e4efff42;
+  --grey-alpha-60: #e4efffa2;
+  --grey-alpha-80: #e5f0ffcc;
+  --blue-alpha-30: #527deb4d;
+  --blue-alpha-60: #c7f8ff99;
+  --blue-alpha-75: #86a5f3bf;
+  --purple-alpha-60: #ddbbff99;
+  --purple-alpha-70: #ab7fd7b3;
```

Comment:
remove all unnecessary variables

Reply:
@Liza-Veis but if we have it dev brach should i remove them too?

Reply:
done